src/routes/index.ts

```diff
@@ -145,6 +145,6 @@ router.get(config.CONFIRMATION_URL, authentication, navigation.hasBOsOrMOs, conf
 router.get(config.UPDATE_LANDING_URL, updateLanding.get);
 
 router.get(config.OVERSEAS_ENTITY_QUERY_URL, authentication, overseasEntityQuery.get);
-router.post(config.OVERSEAS_ENTITY_QUERY_URL, authentication, overseasEntityQuery.post);
+router.post(config.OVERSEAS_ENTITY_QUERY_URL, authentication, ...validator.overseasEntityQuery, checkValidations, overseasEntityQuery.post);
```

Comment:
@jkayyala could you check this [commit](https://github.com/companieshouse/overseas-entities-web/pull/482/commits/47b91012d18901a07c99550c56b05f83fb17cc2d) in UAR-91 branch where I separated out the routes/config for the update journey. As part of your rebase/merge I think you now have both declaration of routes and the validator needs to passed in through the route defs at the bottom of the file